web/css/grouphub.css

```diff
@@ -886,20 +886,21 @@ li.group_section div:nth-child(2) p:first-child {
     color: black;
     padding-top: 10px; }
 /* line 44, _group_section.scss */
-li.group_section div:nth-child(2) p:nth-child(2) {
+li.group_section div:nth-child(2) p.description,
+li.group_section div:nth-child(2) div.extra-group-info p {
     color: #8a8a8a;
     font-size: 13px;
     overflow: hidden;
     white-space: nowrap;
     text-overflow: ellipsis;
 }
 /* line 49, _group_section.scss */
-li.group_section div:nth-child(2) p:last-child {
+li.group_section div:nth-child(2) p.date_and_owner {
```

Comment:
I'm glad you removed the other `:last-child` and `:nth-child()` pseudo-selectors in this PR, yet the `:nth-child(2)` that is still present is not really a robust solution. Would be nice to refactor this in due time.